Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreeTransaction.h

```diff
@@ -401,6 +404,7 @@ class RemoteLayerTreeTransaction {
     bool m_viewportMetaTagWidthWasExplicit { false };
     bool m_viewportMetaTagCameFromImageDocument { false };
     bool m_isInStableState { false };
+    bool m_paintedAnyBackingStore { false }; // Set by buildTransaction but not IPCed.
```

Comment:
Is transaction the right place to hold this flag since it is not going to be encoded? I think WebPage might be a better to have this flag. It is set by RemoteLayerTreeContext and consumed by RemoteLayerTreeDrawingArea and both has access to WebPage.